### ISSUE TITLE
[conan.io/center] Standalone package for Boost.Python?

### ISSUE THREAD

**Issue by sinall** [conan-io/conan-center-index#8147]
Build boost with `boost:without_python=False` takes too long. I am wondering if it leads full compilation. Could you provide a standalone package or Boost.Python? then we don't need to recompile other packages again.

**Comment by SSE4** [comment]
yes, it will result in full re-compilation right now. for modular boost, better ask @grafikrobot

**Comment by jgsogo** [comment]
Hi, @sinall . 

From @bincrafters experience, it is very hard to maintain one different recipe for each boost library, and we decided in CCI to have just one with all `boost` libraries. As Conan lets you reuse the binaries it shouldn't be a problem, **you just need to build your configuration once**... or taking advantage of components, you can get the package with everything and link just the libraries you want to use.
